src/elements/content-sidebar/activity-feed/task-new/Task.js

```diff
@@ -81,6 +82,34 @@ class Task extends React.Component<Props, State> {
         modalError: undefined,
         isEditing: false,
         isLoading: false,
+        isAssigneeListOpen: false,
+    };
+
+    componentDidUpdate(prevProps: Props) {
+        // reset state to props assigned_to
+        const { assigned_to } = this.props;
+        const { assigned_to: prevAssignedTo } = prevProps;
+        if (prevAssignedTo !== assigned_to) {
+            this.setState({
```

Comment:
Is it necessary to copy the prop value into state? If so, should we be using `getDerivedStateFromProps`?